### DISCUSSION TITLE
阿里云盘 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#164] [模板讨论]
阿里云盘 模板问题反馈评论区
有问题可以下面反馈。

**Comment by jjjyyyllll** [comment]
refresh_token必须是抓app上的吗 安卓需要root的吧？

**Reply by wjf0214** [reply]
也可以抓web端的。我做个流程图给你看。
![25P4V00C6J~Z5YP0ZE1{931](https://user-images.githubusercontent.com/23102821/217480773-3a00a77a-036b-4e4b-b46a-1e680ea3b3d7.png)

**Reply by jjjyyyllll** [reply]
失败。。
Failed at 1/6 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 400: Bad Request,
Request URL: api://util/toolbox/notepad

帐号 密码 UA  记事本的ID   里面 阿里云盘的refresh_token值：************  token按照大佬的方法抓的  “refresh_token值：******” 中间有空格

**Reply by jjjyyyllll** [reply]
会不会是 阿里云盘的refresh_token值：************   这个全角 ： 的问题？

**Reply by icaca** [reply]
你这个请求notepad就报错了~

**Reply by jjjyyyllll** [reply]
> 你这个请求notepad就报错了~

不是填“记事本编号:” 这个值吗？   用的1 和 1000  都是报错

**Reply by wjf0214** [reply]
> 会不会是 阿里云盘的refresh_token值：************ 这个全角 ： 的问题？

内容照着说明复制就行。
不过你400错误。你看看你账号密码有没有错误。还有记事本的编号。。
这三个值肯定有一个有问题。。。

**Comment by icaca** [comment]
工具箱 记事本在哪呀... ios上没找到啊

**Reply by wjf0214** [reply]
记事本是在签到框架那里的。不是阿里云盘里面的。

**Reply by icaca** [reply]
感谢感谢🙏

**Reply by icaca** [reply]
大佬帮忙分析分析  可能是什么问题
Failed at 2/6 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 400: Bad Request,
Request URL: https://auth.aliyundrive.com/v2/account/token

**Reply by icaca** [reply]
notepad填写值就行了吧?  
【阿里云盘的refresh_token值：xxxxxxxxxxxxxxxxxxxxxxxx】
冒号前面的应该不用吧?

**Reply by icaca** [reply]
好吧,看了代码真的需要.

**Reply by icaca** [reply]
可以了,太强了,🎉

**Reply by wjf0214** [reply]
【】里面的都需要。
虽然可以不要，但是考虑到后面可能会多个记事本。不加前面的，我会不清楚每个记事本都是干什么用的。所以限制签名的也都需要了。。算是做个备忘记录说明。。

**Reply by jjjyyyllll** [reply]
你的可以了吗？  
帐号是手机号 密码  都没问题 
记事本新建的10086   里面是  阿里云盘的refresh_token值：xxxxxxxxxxxxxxxxxxxxxxxx    新抓的token。。

还是报错

**Reply by jjjyyyllll** [reply]
框架是20230111

**Reply by wjf0214** [reply]
> 你的可以了吗？ 帐号是手机号 密码 都没问题 记事本新建的10086 里面是 阿里云盘的refresh_token值：xxxxxxxxxxxxxxxxxxxxxxxx 新抓的token。。
> 
> 还是报错

账号密码是签到框架的账号密码。
不是阿里云的账号密码。

**Reply by jjjyyyllll** [reply]
> > 你的可以了吗？ 帐号是手机号 密码 都没问题 记事本新建的10086 里面是 阿里云盘的refresh_token值：xxxxxxxxxxxxxxxxxxxxxxxx 新抓的token。。
> > 还是报错
> 
> 账号密码是签到框架的账号密码。 不是阿里云的账号密码。

啊。哈哈哈。 原来是这样。感谢大佬，成功了

**Comment by gitcps** [comment]


**Reply by wjf0214** [reply]
ua首页有说明。还不太清楚的话。可以搜一下是什么东西。
qd账号密码记录信息。不用经常更新参数。

**Reply by 121592512** [reply]
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36


ua 是这个吗

**Comment by 121592512** [comment]
![image](https://user-images.githubusercontent.com/36238018/223372732-001417ac-8c4d-41c6-8925-a3731038fbba.png)
![image](https://user-images.githubusercontent.com/36238018/223372847-f60508ac-fb51-47fe-a106-f129b205d8b0.png)
设置如图

**Reply by wjf0214** [reply]
有什么问题？

**Reply by 121592512** [reply]
> 有什么问题？

![image](https://user-images.githubusercontent.com/36238018/223376774-04a872e0-0375-401b-ac32-5fa630ba8414.png)
bao'cu报错。  但是我检查了， 账号密码是签到框架的账号密码。， 记事本编号5 6  7 试了三个都一样。

**Reply by wjf0214** [reply]
如果确定信息没填错的话。找到对应任务，点击模板手动执行一下第一个请求。看下报错信息。

**Comment by sohaoo** [comment]
2023-04-07 15:24:00 
日志：四月挑战:累计总共签到7天

------
终于成功了，总结下：
1.阿里云盘的refresh_token值： 最后这个冒号是全角，全角，全角，楼主的正则筛选如果输入半角是筛不出来的
2.refresh_token不是抓电脑上的cookie，电脑访问 https://alist.nn.ci/zh/guide/drivers/aliyundrive.html ，然后用阿里APP扫描登陆后再刷新出来的token才是
3.账号，密码是签到的账号密码

祝大家顺利，感谢楼主~

**Comment by chen8945** [comment]
前几天签到奖励改为了手动领取，需要在签到后面新增一个请求，如下图
![image](https://user-images.githubusercontent.com/44148812/233822696-460e8992-2323-4a55-ad5d-75529e56d6f1.png)
![image](https://user-images.githubusercontent.com/44148812/233822708-9db2b1ae-365d-40cd-9933-73c50b3419bd.png)

**Reply by wjf0214** [reply]
已修改。

**Comment by gitcps** [comment]
现在签到奖励有1/2/3个，但是领取只能领取一个奖励

**Reply by windyhun** [reply]
奖励大头都在2上面

**Comment by fenghuang111** [comment]
![iShot_2023-12-22_17 58 35](https://github.com/wjf0214/qd-templates/assets/29788626/9c3fd2d6-5904-4c2d-9606-7dffff5b4387)
bao 这个错

**Comment by windyhun** [comment]
更新下吧，任务2任务3大多数是已经完成的。

**Comment by hecarli555** [comment]
一直签到都是好好的，今天突然签到失败了，提示如下
Failed at 6/7 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 403: Forbidden,
Request URL: https://member.aliyundrive.com/v1/activity/sign_in_reward

**Reply by zzzdrv** [reply]
同上：

2024-05-21 15:04:05
日志：Failed at 6/7 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 403: Forbidden,
Request URL: https://member.aliyundrive.com/v1/activity/sign_in_reward

**Comment by hecarli555** [comment]
有没有大佬解答一下？？？？是不是账号被限制了？

**Reply by icaca** [reply]
网页端好像不让签到了~

**Comment by zcb411711** [comment]
今天全部挂掉了，重新弄也不管用

**Comment by truynal** [comment]
大佬用不了了；

2024-05-22 17:36:58
日志：Failed at 6/7 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 403: Forbidden,
Request URL: https://member.aliyundrive.com/v1/activity/sign_in_reward

**Reply by truynal** [reply]
网页端签到接口还在，点了还可以签到
<img width="290" alt="截屏2024-05-22 17 41 20" src="https://github.com/wjf0214/qd-templates/assets/98015467/cad99f3c-30a0-42a4-830a-c2271587b9d5">

**Reply by icaca** [reply]
你点一下 会提示到桌面版签到

**Reply by truynal** [reply]
我点一下就已经签成功了

**Reply by icaca** [reply]
奇怪 我早晨还点了的 ，我明天再试试

**Reply by icaca** [reply]
https://github.com/wjf0214/qd-templates/assets/279229/b36cdc61-6329-4a07-a1f4-0fa2d4d19dca

**Reply by truynal** [reply]
……看左下角，你点那是其他活动

**Reply by icaca** [reply]
这个页面是点左下角弹出来的，我明天再试试~

**Reply by truynal** [reply]
仔细看，你已经签到成功

**Comment by Luci8109** [comment]
2024.5.21阿里更新了请求头，用不了了，麻烦大佬更新下，感谢~

**Comment by jjjyyyllll** [comment]
求大佬更新，现在版本不能用了

**Comment by tasty007** [comment]
2024.5.21阿里更新了请求头，用不了了，麻烦大佬更新下，感谢~

**Comment by hecarli555** [comment]
2024.5.21阿里更新了请求头，用不了了，麻烦大佬更新下，感谢~

**Comment by Andirxiao** [comment]
2024.5.21阿里更新了请求头，用不了了，麻烦大佬更新下，感谢~

**Comment by hecarli555** [comment]
阿里云盘签到还会修复不？

**Comment by hecarli555** [comment]
大佬还会修复吗？？？最近大佬潜水都不回复了。。。。
